doc/api/util.md

```diff
@@ -1631,6 +1631,71 @@ const module = new WebAssembly.Module(wasmBuffer);
 util.types.isWebAssemblyCompiledModule(module);  // Returns true
 ```
 
+### util.createPromiseHook(hook)
+<!-- YAML
+added: REPLACEME
+-->
```

Comment:
I think this should be flagged as experimental.

Reply:
good idea